thorlcr/activities/msort/thmsortslave.cpp

```diff
@@ -156,10 +160,14 @@ class MSortSlaveActivity : public CSlaveActivity
         }
         if (hasStarted())
         {
-            ActPrintLog("SORT waiting barrier.2");
+            if (traceActivity())
+                ActPrintLog("SORT waiting barrier.2");
             barrier->wait(false);
-            ActPrintLog("SORT barrier.2 raised");
-            ActPrintLog("SORT waiting for merge");
+            if (traceActivity())
+            {
+                ActPrintLog("SORT barrier.2 raised");
```

Comment:
these types of messages have often been useful in identifying where and which workers are stalled.. probably not the best way, but will probably make it harder to diagnose which/where without them.
Maybe we should be logging, but not to logging backend, but only to postmortem log.